akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -0,0 +1,93 @@
+* [Migrating Untyped to Typed](#migrating-untyped-to-typed)
+* [Cluster Shutdown](#cluster-shutdown)
+* [Cluster Configuration Compatibility Check](#cluster-configuration-compatibility-check)
+ 
+## Serialization Compatibility
+
+There are two parts of Akka that need careful consideration when performing an rolling update.
+
+1. Compatibility of remote message protocols. Old nodes may send messages to new nodes and vice versa.
+1. Serialization format of persisted events and snapshots. New nodes must be able to read old data, and
+   during the update old nodes must be able to read data stored by new nodes.
+
+There are many more application specific aspects. It's important to have a strategy for serialization
+format that can be evolved and you can find advice in
+@ref:[Persistence - Schema Evolution](../persistence-schema-evolution.md), which also applies to
+remote messages when deploying with rolling updates.
+
+### Retiring a serializer without downtime
+
+One approach is carried out in two phases: 
+
+1. Deploy a version of your application that still supports deserializing messages in the old form 
+(your deserializer is still in "akka.actor.serializers"-section of your configuration, but it no longer serializes new messages 
+(it is removed from “akka.actor.serialization-bindings”-section). See @ref:[Serialization Configuration](../serialization.md#configuration) for details.
+1. After this upgrade has succeeded and there are no longer systems sending messages serialized with the decommissioned serializer, 
+you can upgrade to a version that also no longer contains this serializer in `akka.actor.serializers`.
+
+## Cluster Sharding
+
+During a rolling upgrade, sharded entities receiving traffic may be moved during @ref:[shard rebalancing](../cluster-sharding.md#shard-rebalancing), 
+to an old or new node in the cluster, based on the pluggable allocation strategy and settings.
+When an old node is stopped the shards that were running on it may be allocated to one of the
+other old nodes remaining in the cluster. 
+ 
+It is recommended to leave the oldest node until last, because the shard coordinator 
+is running on that node. See @ref:[Cluster Shutdown](#cluster-shutdown) below for more detail.
+
+There are some cases when @ref:[a full cluster restart is needed](../cluster-sharding.md#rolling-upgrades).
+
+## Migrating Untyped to Typed
+
+It is recommended with a two step approach:
+
+* Deploy with the new nodes set to `akka.cluster.configuration-compatibility-check.enforce-on-join = off`
+and ensure all nodes are in this state
+* Deploy again and with the new nodes set to `akka.cluster.configuration-compatibility-check.enforce-on-join = on`. 
+
+The configuration from existing nodes should pass the @ref:[Cluster Configuration Compatibility Checks](#cluster-configuration-compatibility-check).
+Find out more about coexisting and @ref:[untyped to typed](../typed/coexisting.md#untyped-to-typed). 
+
+### With Cluster Sharding and Persistence
```

Comment:
This section related to and could be updated with #27342. And samples from @patriknw's work in akka/akka-samples#110 linked when merged.